src/app/services/reducers/auth-reducers.ts

```diff
@@ -0,0 +1,12 @@
+import { IAction } from '../../../types/action';
+import { AUTHENTICATE_USER } from '../constants';
+
+export function authResponse(state = {}, action: IAction) {
+    switch (action.type) {
+        case AUTHENTICATE_USER:
+        localStorage.setItem('authenticated', JSON.stringify(action.response));
```

Comment:
Indent the statements inside case.

Reducers are supposed to be pure functions. A pure function is a function that doesn't have side effects. Example of side effects are network requests, querying a database etc. Accessing local storage here is a side effect. You shouldn't access local storage here, you can do it in an action creator.

Reply:
makes sense. adding a new action creator